install_config/aggregate_logging.adoc

```diff
@@ -937,6 +949,63 @@ existing `journald` log driver configuration. It is recommended to use the
 Logging Driver] for instructions to change your existing log driver
 configuration to `json-file`.
 
+[[fluentd-file]]
+*Configuring Fluentd to Send Logs to a File*
+
+Fluentd writes logs to a file specified by the `LOGGING_FILE_PATH` environment variable, by default `/var/log/fluentd/fluentd.log`. 
+
+Use the *logs* utility to print out the contents of Fluentd log files. For example:
+
+----
+oc exec <pod> logs <1>
+----
+
+<1> Specify the name of the Fluentd or Elasticsearch pod.
+
+For example:
+
+----
+oc exec logging-es-data-master-2-deploy logs
+oc exec logging-kibana-1 logs
+----
+
+The contents of log files are printed out, starting with the oldest log.  Use `-f` option to follow what is being written into the logs.
+
+To control the size of the logs, you can specify a maximum log size and the length of time that {product-title} retains the logs. 
+ 
+* The `LOGGING_FILE_SIZE` parameter specifies how large the file can be before {product-title} rotates the files. When the log reaches the specified size, {product-title} automatically compresses and renames the *fluentd.log* log file so that new logging data can be collected.  
+
+* The `LOGGING_FILE_AGE` parameter specifies how many of the compressed log files can be retained before {product-title} deletes the files. 
+
+The following example shows logs in a cluster where `LOGGING_FILE_SIZE=1Mb` and `LOGGING_FILE_AGE=4`. When the *fluentd.log* reaches 1Mb, {product-title} deletes the current *fluentd.log.4*, renames the *fluentd.log* and *fluentd.log.1* and renames the other files in turn.  
+
+----
+fluentd.log     0b
+fluentd.log.1  1Mb
+fluentd.log.2  1Mb
+fluentd.log.3  1Mb
+fluentd.log.4  1Mb
+----
+
+.Parameters for configuring Fluentd logging
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `LOGGING_FILE_PATH` | Speciy the path to the Fluentd log file. If you want Fluentd to output to the console, `STDOUT`, set this variable to `console`. The default value is `/var/log/fluentd/fluentd.log`.
+| `LOGGING_FILE_AGE` | The number of logs that Fluentd retains before deleting. The default value is `10`.
+| `LOGGING_FILE_SIZE` | The number of log files that are retained before {product-title} deletes the files. If the number of the log file exceeds this value, the log file gets rotated. The default is 1MB.
```

Comment:
> The number of log files that are retained before {product-title} deletes the files. If the number of the log file exceeds this value, the log file gets rotated. The default is 1MB.

Can you replace this text by the following?:

Is a maximum size of a single log file in Bytes. If this value is exceeded by `fluentd.log`, log files are renamed accordingly and new `fluentd.log` is created. If there are more log files than is a value of `LOGGING_FILE_AGE` the oldest log gets deleted (Log rotation). Default value is `1024000` which means it is 1 MB.

Reply:
Well, if it makes sense to an English speaker.